README.md

```diff
@@ -55,9 +55,8 @@ Create an index.html file (see [docs](/docs/root-application.md#indexhtml-file)
 Create the root application (see [docs](/docs/root-application.md) for more detail).
 ```js
 // src/main.js
-import { declareChildApplication, setLoader, start } from "single-spa";
-setLoader(SystemJS);
-declareChildApplication('/apps/app1/app1.js', () => window.location.hash === '');
+import { declareChildApplication, start } from "single-spa";
```

Comment:
The changes here to the Readme are just really minor ones that will be followed up with a docs rework